src/App.tsx

```diff
@@ -34,16 +34,7 @@ function App() {
           <Route path="/profile" element={<UserProfileScreen/>} />
           <Route path="/interests" element={<Interests/>} />
           <Route path="/portal" element={<Portal/>} /> 
-          <Route path="/getresources" element={<GetResources/>} />           
-          <Route path="/login" element={<Login />} />
-          <Route path="/signup" element={<Register />} />
-          <Route path="/home" element={<Home />} />
-          <Route path="/" element={<Welcome />} />
-          <Route path="/address-info" element={<AddressInfo />} />
-          <Route path="/address-entry" element={<AddressVerify />} />
-          <Route path="/profile" element={<UserProfileScreen />} />
-          <Route path="/interests" element={<Interests />} />
```

Comment:
I see you removed these routes. Are they simply not used? Were they replaced somewhere else?

Reply:
They are duplicated, lines above are same routes